src/openapi.ts

```diff
@@ -106,7 +87,10 @@ export const toOpenAPISpec = (
             in: 'query',
             name,
             description: schema.description,
-            schema: { type: getSchemaObjectType(schema) },
+            // @ts-expect-error TS detects a mismatch between the JSONSchema types
+            // between openapi-types and json-schema. Ignore and assume everything
+            // is cool.
```

Comment:
This directive + comment is repeated in several places in this file.

Reply:
This is a way.

It will ignore the case like `type: any`?

What happened when `type` is not declared like the test case showed before? I don't know if it breaks any existing `openapi.yaml` (guess we will see)